apps/clinic/src/modules/patients/patients.consumer.ts

```diff
@@ -1,22 +1,17 @@
 import { Controller } from '@nestjs/common';
-import {
-  KafkaContext,
-  MessagePattern,
-  Ctx,
-  Payload,
-} from '@nestjs/microservices';
+import { MessagePattern, Payload } from '@nestjs/microservices';
 
-import { IKafkaMessage } from '@vp-clients-app/common-pkg';
+import { EventsEnum, IKafkaMessage } from '@vp-clients-app/common-pkg';
 
 import { PatientEntity } from 'src/modules/patients/patient.entity';
 import { PatientsService } from 'src/modules/patients/patients.service';
 import { PatientCreateDto } from 'src/modules/patients/dtos/patient.dto';
 
 @Controller('patients')
-export class PatientsMessagesController {
+export class PatientsConsumer {
```

Comment:
I would suggest renaming it as PatientsKafkaConsumer, because it is not clear what exactly is being consumed, and this consumer is not applicable for other types of transports